recipe/meta.yaml

```diff
@@ -28,9 +32,11 @@ test:
         - test -f $PREFIX/lib/libflann_s.a                                        # [unix]
         - test -f $PREFIX/lib/libflann.dylib                                      # [osx]
         - test -f $PREFIX/lib/libflann.so                                         # [linux]
+        - test -f $PREFIX/lib/libflann.dll                                        # [win]
         - test -f $PREFIX/lib/libflann_cpp_s.a                                    # [unix]
         - test -f $PREFIX/lib/libflann_cpp.dylib                                  # [osx]
         - test -f $PREFIX/lib/libflann_cpp.so                                     # [linux]
+        - test -f $PREFIX/lib/libflann_cpp.dll                                    # [osx]
```

Comment:
I have no idea, because I don't have access to a windows box other than AppVeyor. I figured this check would tell me whether the files exist :smile: